source-map-support.js

```diff
@@ -21,6 +21,10 @@ function isInBrowser() {
   return ((typeof window !== 'undefined') && (typeof XMLHttpRequest === 'function'));
 }
 
+function hasGlobalProcess() {
```

Comment:
`hasGlobalProcessEventListener()`?

It's far from pretty but IMHO it is a bit clearer.